packages/story-editor/src/components/library/panes/media/common/stories/mediaGallery.js

```diff
@@ -290,9 +293,29 @@ const Container = styled.div`
 `;
 
 export const _default = (args) => {
+  const mediaValue = {
+    local: {
+      state: {
+        isCurrentResourceProcessing: () => false,
+        isCurrentResourceUploading: () => false,
+      },
+      actions: {
+        uploadMedia: noop,
+      },
+    },
+  };
+
   return (
     <Container>
-      <MediaGallery resources={resources} providerType={'unsplash'} {...args} />
+      <MediaContext.Provider value={mediaValue}>
+        <CanvasProvider>
+          <MediaGallery
+            resources={resources}
```

Comment:
Do the resources all have the same keys? Let's fix that :) 

![image](https://user-images.githubusercontent.com/22185279/153429794-aa26753c-c2df-49c7-a20d-02ee707a928b.png)

Reply:
>Do the resources all have the same keys? Let's fix that :)

Yes they are. I had updated them along with some attempts to fix, but when those fixes didn't work I reverted and completely forgot about that. Nice catch!